Tools/tsan/suppressions_free_threading.txt

```diff
@@ -43,6 +43,10 @@ race_top:write_thread_id
 race_top:PyThreadState_Clear
 # Only seen on macOS, sample: https://gist.github.com/aisk/dda53f5d494a4556c35dde1fce03259c
 race_top:set_default_allocator_unlocked
+# See https://github.com/python/cpython/issues/128421
+race_top:_PyFrame_GetBytecode
+race_top:BaseException_set_tb
```

Comment:
I'm not able to reproduce any races in `Lib/test/test_asyncio/test_free_threading.py` with or without these suppressions.

The linked issue looks like it shows a race when running faulthandler on a segfault, but:

1) The tests shouldn't have segfaults!
2) I think that issue is now fixed by GH-128425

Would you please look into this again and either update the issue or remove the added suppressions?

Reply:
I am also unable to reproduce the race now kinda weird how it was happening before, I have removed the suppressions from PR but I'll keep the issue open for some more time incase I reproduce it later.